src/api/routes.ts

```diff
@@ -48,6 +48,13 @@ export type ItemSearchParams =
       types?: UnionOfConst<typeof ItemType>[];
     }
   | undefined;
+export type ItemChildrenParams =
+  | {
+      ordered?: boolean;
+      types?: UnionOfConst<typeof ItemType>[];
+    }
+  | undefined;
```

Comment:
I would prefer to mark the key in the object as optional.

Reply:
Looking at the rest I think you should remove the undefined here, since you will be injecting the `ordered` key for all calls below the hook.